website/site/content/docs/authentication-backends.md

```diff
@@ -137,16 +146,14 @@ Check each project's documentation for instructions on how to configure it.
 
 ## Options
 
-The `git-gateway`, `github`, and `gitlab` backends all allow some additional optional fields for certain use
-cases. A full reference is below. Note that these are properties of the `backend` field, and should
-be nested under that field.
+Netlify CMS backends allow some additional fields for certain use cases. A full reference is below. Note that these are properties of the `backend` field, and should be nested under that field.
 
 | Field           | Default                                                        | Description                                                                                                                                          |
 | --------------- | -------------------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------- |
-| `repo`          | none                                                           | **Required** for `github` and `gitlab` backends; ignored by `git-gateway`. Follows the pattern `[org-or-username]/[repo-name]`.                                    |
+| `repo`          | none                                                           | **Required** for `github`, `gitlab`, and `bitbucket` backends; ignored by `git-gateway`. Follows the pattern `[org-or-username]/[repo-name]`.                                    |
 | `accept_roles`  | none                                                           | `git-gateway` only. Limits CMS access to your defined array of user roles. Omitting this field gives access to all registered users.                 |
 | `branch`        | `master`                                                       | The branch where published content is stored. All CMS commits and PRs are made to this branch.                                                       |
-| `api_root`      | `https://api.github.com` (GitHub) or `https://gitlab.com/api/v4` (GitLab)  | The API endpoint. Only necessary in certain cases, like with GitHub Enterprise or self-hosted GitLab.                                                                      |
+| `api_root`      | `https://api.github.com` (GitHub), `https://gitlab.com/api/v4` (GitLab), or `https://api.bitbucket.org/2.0` (Bitbucket)  | The API endpoint. Only necessary in certain cases, like with GitHub Enterprise or self-hosted GitLab.                                                                      |
```

Comment:
I kept the Bitbucket mentions in the defaults, but removed any mention of self-hosted Bitbucket.